### DISCUSSION TITLE
How to enable captureStackTrace?

### DISCUSSION THREAD

**Discussion by DanielSOhara27** [gajus/slonik#388] [Q&A] [answered]
Hi, 

I've been use slonik for some developments of mine and have been very happy with this tool. When I started using you were on version v28.1.0 and now you are at v30.x. I saw in your release notes that there are only 2 breaking changes between v28.x and v30.x, but I have a question regarding the `captureStackTrace default value`. 

Since it is no longer defaulted to true, how could we turn it on if we need it? would we need to go into the `node_module source code` and change it back, or is there a constructor/property we can change? How and where would we go about interacting with that `captureStackTrace` property?

Thanks in advance and I hope it helps others.

**Comment by gajus** [comment, marked as answer]
It is a property of the client configuration.

https://github.com/gajus/slonik#api
